src/main/java/org/example/fileManager/FileManager.java

```diff
@@ -13,32 +12,41 @@ public class FileManager {
     private FiletypeStrategy selectFiletype(int file) {
         switch (file) {
             case 1 -> {
-                return new CsvFile();
+                return new CsvFileManager();
             }
             case 2 -> {
-                return new JsonFile();
+                return new JsonFileManager();
             }
             case 3 -> {
-                return new XlsxFile();
+                return new XlsxFileManager();
             }
             case 4 -> {
-                return new XmlFile();
+                return new XmlFileManager();
             }
             case 5 -> {
-               return new YamlFile();
+               return new YamlFileManager();
             }
             default -> System.out.println("Invalid filetype!");
         }
         return null;
     }
 
     public void selectOperation(int file, int operation, int data) throws Exception {
```

Comment:
Operation też można zrobić na enumie, choć tak na prawdę to powinny być dwie osobne metody